test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -234,7 +242,7 @@ json_spirit::mObject fillBCTest(json_spirit::mObject const& _input)
 	size_t importBlockNumber = 0;
 	string chainname = "default";
 	string chainnetwork = "default";
-	std::map<string, ChainBranch*> chainMap = { {chainname , new ChainBranch(genesisBlock)}};
+    std::map<string, ChainBranch*> chainMap = { {chainname , new ChainBranch(genesisBlock, miningType)}};
```

Comment:
Never use raw pointers. Use `unique_ptr` here.
Extra space before comma.

Reply:
not for this PR